addons/account_payment/views/account_portal_templates.xml

```diff
@@ -60,6 +60,14 @@
     </template>
 
     <template id="portal_invoice_page_inherit_payment" name="Payment on My Invoices" inherit_id="account.portal_invoice_page">
+        <xpath expr="//t[@t-call='portal.portal_record_sidebar']//h2[1]" position="after">
+            <div t-if="invoice.payment_state in ('paid', 'in_payment')" class="position-sm-absolute position-lg-relative me-auto me-lg-0 top-0 end-0 rounded text-bg-success fs-6 fw-normal badge">
+                <i class="fa fa-fw fa-check-circle"/> Paid
+            </div>
+            <div t-else="pending_txs" class="position-sm-absolute position-lg-relative me-auto me-lg-0 top-0 end-0 rounded text-bg-info fw-normal fs-6 badge" >
```

Comment:
I assume you intended to use `t-elif` rather than `t-else`, as `t-else` doesn't care about the provided value.

Reply:
If so, you should define what is `pending_txs` because it currently evaluate as `None`.

Reply:
Good catch. The issue was more the fact we called the same element twice but with a different styling